administrate/zone-migration.md

```diff
@@ -0,0 +1,120 @@
+
+One of the first things you can do is to lower your DNS records TTL (time to live) to speed up the DNS propagation when you will update your domains DNS records.
+
+On your registrar's interface, find your application's domain. Usually, the DNS record will be `domain.tld IN CNAME domain.<current-zone>.clever-cloud.com` with a certain TTL.
+Update that TTL to 60 seconds instead and wait for this change to propagate (wait as much time as the old TTL value). This change will allow a faster DNS propagation when
+you will have to update the `CNAME` record value to `domain.<target-zone>.clever-cloud.com`, lowering the amount of time your users will experience an increased latency.
+
+{{< alert "info" "Note" >}}
+After the migration, you can set back your old TTL value.
+{{< /alert >}}
+
+#### Increased latency
+
+Once your application has been redeployed to your target zone, your application's domain will still have its DNS pointing to the old zone. As a consequence, it
+may increase the latency for your users. For example, if you are migrating from Paris (PAR) to Montreal (MTL), your visitors will connect to our Paris infrastructure that will
+then redirect the trafic on our Montreal infrastructure, leading to requests taking more time.
+
+You might also experience an increased latency between your application and other services (like a PostgreSQL add-on for example). If your application has switched to the Montreal zone
+and your PostgreSQL add-on is still in Paris, your application will need extract time to query the database.
+
+#### DNS updates
+
+Once your application has been migrated to your target zone, you will find in the `Domain names` section the new `A` and `CNAME` DNS records to use for your application's domains.
+If you were using `A` records, update the records (there might be more or less than what you had previously, this is normal). If you used a `CNAME` record, simply change the value.
+
+Once done, it will take as much time to propagate as to what value was defined your TTL for that record. If you followed the prerequisites above, it should take a few minutes tops.
+
+### Migration
+
+If your application doesn't match any criteria of the list in the `Application migration` section, then you can go to the `Information` pane of your application and under the `Zone` label, chose your target zone.
+Once selected, you can save your changes using the `Save` button at the bottom of the page.
+
+After saving, you can now redeploy your application in the `Overview`, it will redeploy on the new zone.
+
+### Applications using an FSBucket
+
+FSBuckets can only be mounted by applications on the same zone. This means that you can' use a FSBucket add-on hosted on Paris if your application is on Montreal.
+
+To migrate FSBuckets, see the [section below](#fsbucket).
+
+### PHP Applications
+
+Due to technical reasons, a PHP application can't be migrated to another zone. In this case, you will have to re-create the application.
```

Comment:
```suggestion
Because of technical reasons, a PHP application can not be migrated to another zone. In this case, you will have to re-create the application.
```